build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
If this is building the lambda zip archive I think you will also need to `touch podaac/__init__.py` before putting it in the zip. This is because lambda doesn't understand namespace packages (which podaac is) and requires it to be a module instead. But putting an `__init__.py` actually in the source podaac/ directory breaks a bunch of imports.

Reply:
Added in [6d64072](https://github.com/podaac/swodlr-ingest-to-sds/pull/1/commits/6d64072f4a945dc43f5639a70ff04a302b8143d8)